sema/ro/creator/strategy/registry.py

```
@@ -0,0 +1,54 @@
+from contextlib import contextmanager
+from typing import Generator
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)